lua_examples/pcm/play_file.lua

```diff
@@ -0,0 +1,40 @@
+-- ****************************************************************************
+-- Play file with pcm module.
+--
+-- Upload jump_8k.u8 to spiffs before running this script.
+--
+-- ****************************************************************************
+
+
+function cb_drained(d)
+  print("drained "..node.heap())
+
+  file.seek("set", 0)
+  -- uncomment the following line for continuous playback
+  --d:play(pcm.RATE_8K)
+end
+
+function cb_stopped(d)
+  print("playback stopped")
+  file.seek("set", 0)
+end
+
+function cb_paused(d)
+  print("playback paused")
+end
+
+file.open("jump_8k.u8", "r")
+
+drv = pcm.new(pcm.SD, 1)
```

Comment:
Nope, that won't fail. The pcm code bases solely on the platform layer where components are linked when referenced. That's independent of `user_modules.h` where the *user interface* on Lua layer is tailored.
Users are free to enable the sigma-delta module, though it's neither required nor recommended to access the hardware directly while it's in use by pcm.